File: cpp-programs/loops/lcm-hcf.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
int a,b,g,hcf,i,lcm;

cout<<"Enter two number :::";
cin>>a>>b;

if(a > b)
 g = a;
else
 g = b;

for(i=1;i<=(a*b);i++)
{
 if((i%a==0)&&(i%b==0))
 {
  lcm = i;
  break;
 }
}

cout<<" the lcm of given number is = "<<lcm<<endl;

hcf =(a * b)/lcm;

cout<<" the hcf of given number is = "<<hcf<<endl;

}

File: java-programs/loops/lcm-hcf.java
import java.io.*;
import java.util.Scanner;

class lcm_hcf
{      
    public static void main(String args[])
    {
     int a,b,i,g, hcf = 0,lcm = 0;
     Scanner sc= new Scanner(System.in);

     System.out.print("Enter first number : ");
     a = sc.nextInt();
     System.out.print("Enter second number : ");
     b = sc.nextInt();

     if(a>b)
     g = a;
     else 
     g = b;

     for(i = g; i<=(a*b); i++)
     {
        if((i%a == 0) && (i%b == 0))
        {
            lcm = i;
            break;
        }
     }

     System.out.println("Hcf : "+lcm);
     hcf =(a * b)/lcm;
     System.out.println("Lcm : "+hcf);
    }
}